src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -105,6 +106,36 @@ def wait_process(proc, wait_time):
             i = i - 1
         return return_code
 
+    def check_files(self, glob_pattern, count=1, retries=10):
+        """Check for files."""
+
+        file_list = glob.glob(glob_pattern)
+        found_files = False
+
+        retry = 0
+        while retry < retries:
+            retry += 1
+            file_list = glob.glob(glob_pattern)
+
+            self.log.info("Found completion files: [%s]\n",
+                          ", ".join(file_list))
+
+            if len(file_list) == count:
+                found_files = True
+                break
+
+            time.sleep(1)
+
+        if not found_files:
+            self.log.info("Expected {} completion files, ".format(str(count)))
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions